pkg/registry/deployment.go

```diff
@@ -231,6 +233,23 @@ func GenerateDeployment(cr *registryv1alpha1.DevfileRegistry, scheme *runtime.Sc
 					corev1.ResourceMemory: resource.MustParse("256Mi"),
 				},
 			},
+			Env: []corev1.EnvVar{
+				{
+					Name:  "NEXT_PUBLIC_ANALYTICS_WRITE_KEY",
+					Value: cr.Spec.Telemetry.RegistryViewerWriteKey,
+				},
+				{
+					Name: "DEVFILE_REGISTRIES",
+					Value: fmt.Sprintf(`
+					[
+						{
+							"name": "Community",
+							"url": "http://localhost:8080",
```

Comment:
should the `url` field be equal to `fqdn`  i.e. set to `cr.Status.URL`.  I'm not sure which one is preferred

Reply:
I believe the registry viewer treats `url` as the internal one it uses to access the index server from within the cluster.